README.md

```diff
@@ -0,0 +1,43 @@
+# OpenTelemetry Auto Instrumentation
+
+This project provides [OpenTelemetry] instrumentation for [Go] libraries using [eBPF].
+
+## Project Status
+
+:construction: This project currently a work in progress.
+
+### Compatability
+
+This project ensures compatibility with the current supported versions of the [Go language](https://golang.org/doc/devel/release#policy):
+
+> Each major Go release is supported until there are two newer major releases.
```

Comment:
Are we sure we want to do this? I think one of the big wins in this project is being able to instrument any Go binary, I think some people for sure have binaries with older Go versions that they can't control. I didn't see this limitation on other auto instrumentation projects. On the other hand, supporting only Go 1.17 and above is much easier. (Go changed their ABI in this version, so no need to support both stack-based and registry-based)

Reply:
We have the same policy for .NET auto instrumentation. The main reason is maintenance costs.

Reply:
This is also the policy of https://github.com/open-telemetry/opentelemetry-go. If we are trying to inter-op with that project I think having a similar compatibility project is appropriate.

For what it's worth, I think this is a good way to start the project off. We can always make this more comprehensive and cover more versions as the project matures if we feel it is needed.